.changeset/few-dingos-happen.md

```diff
@@ -0,0 +1,12 @@
+---
+'@clerk/localizations': minor
+'@clerk/clerk-js': minor
+'@clerk/types': minor
+---
+
+Add a confirmation input as an additional check when doing destructive actions such as:
+- delete an organization
+- delete a user account
+- leave an organization
+
+New localization key was added(organizationProfile.leaveOrganization.actionDescription)
```

Comment:
Should we mention all new keys here?

Reply:
It'd probably be overkill :) Just state that "new keys were introduced to support the above"

Reply:
@nikosdouvlis we also removed one key(see [here](https://github.com/clerkinc/javascript/pull/1454/files#:~:text=description%3A%20LocalizationValue%3B)) to be consistent with the other screens.

Reply:
Is this UI final? 
Can I get a link to the figma design please?